docs/developer/guide.md

```diff
@@ -20,3 +20,10 @@ The following steps are needed to introduce a new resource and its respective re
 - Reference the new resource in [pkg/options/resource.go](https://github.com/kubernetes/kube-state-metrics/blob/master/pkg/options/resource.go).
 - Add a sample Kubernetes manifest to be used by tests in the [tests/manifests/](https://github.com/kubernetes/kube-state-metrics/tree/master/tests/manifests) directory.
 - Lastly, and most importantly, actually implement your new resource(s) and its test binary in [internal/store](https://github.com/kubernetes/kube-state-metrics/tree/master/internal/store). Follow the formatting and structure of other resources.
+
+### Common test failures
+
+1. File is not `goimports`-ed with -local k8s.io/kube-state-metrics (goimports)
+```
+find . -name "*.go" ! -path "./vendor/*" | xargs goimports -local "k8s.io/kube-state-metrics/v2" -w
```

Comment:
Should we recommend running `make lint` instead? We have local packages configured here: https://github.com/kubernetes/kube-state-metrics/blob/master/.golangci.yml#L23

Reply:
`make lint` shows lint errors while doesn't fix these errors.

I run `goimports` but still got this lint error, then I find need to add `-local k8s.io/kube-state-metrics/v2`.

Reply:
Makes sense, we can add `lint-fix` to the `Makefile`.